language-support/java/codegen/BUILD.bazel

```diff
@@ -90,22 +91,32 @@ daml_lf_target_versions = ["1.0", "1.1"]
 ####  Integration Tests
 ########################################################
 [
-    daml_to_java(
-        name = "integration-tests-model-%s" % target,
-        main_src = "src/it/daml/Lib.daml",
-        srcs = glob(["src/it/daml/**/*.daml"]),
-        target = target,
-    )
+    [
+        daml_to_java(
+            name = "integration-tests-model-noprefix-%s" % target,
```

Comment:
Please add a comment why we're compiling the test models with and without prefix.